Duplicati/Library/Backend/Jottacloud/Jottacloud.cs

```diff
@@ -222,6 +253,63 @@ public IEnumerable<IFileEntry> List()
             }
         }
 
+        /// <summary>
+        /// Retrieves info for a single file (used to determine file size for chunking)
+        /// </summary>
+        /// <param name="remotename"></param>
+        /// <returns></returns>
+        public IFileEntry Info(string remotename)
+        {
+            var doc = new System.Xml.XmlDocument();
+            try
+            {
+                // Send request and load XML response.
+                var req = CreateRequest(System.Net.WebRequestMethods.Http.Get, remotename, "", false);
+                var areq = new Utility.AsyncHttpRequest(req);
+                using (var rs = areq.GetResponseStream())
+                    doc.Load(rs);
+            }
+            catch (System.Net.WebException wex)
+            {
+                if (wex.Response is System.Net.HttpWebResponse && ((System.Net.HttpWebResponse)wex.Response).StatusCode == System.Net.HttpStatusCode.NotFound)
+                    throw new FileMissingException(wex);
+                throw;
+            }
+            // Handle XML response. Since we in the constructor demand a folder below the mount point we know the root
+            // element must be a "folder", else it could also have been a "mountPoint" (which has a very similar structure).
+            // We must check for "deleted" attribute, because files/folders which has it is deleted (attribute contains the timestamp of deletion)
+            // so we treat them as non-existant here.
+            var xFile = doc.DocumentElement;
+            if (xFile.Attributes["deleted"] != null)
+            {
+                throw new FileMissingException();
```

Comment:
It might be helpful to provide an exception message here to help in debugging.

Reply:
I'm not sure what is considered helpful in this case. File name, url, xml body? I see DropBox returns some json here.

Reply:
I've added remotename to the exception. Also de-duplicated some xml handling code.

Reply:
I'm not sure.  Hopefully someone that's more familiar with the backend implementations can help out here.

Reply:
The way I have handled this in other backends, like GCloud etc, is to simply keep a cache of items in the backend. This is required as each item has a GUID of sorts, which is not related to the filename, so even though the API requests something by name, we need to translate it into a GUID. To avoid repeated (recursive) lookups, I opted to cache them.

If you need something similar, you can do the same, but we can also consider it there is a better way. Not immediately sure what that way would be though, as there are case where the filename is all we know (i.e. we did not call `List()` on before calling `Get()`).